### DISCUSSION TITLE
How do I ignore a table when generating types?

### DISCUSSION THREAD

**Discussion by karloluis** [kristiandupont/kanel#442] [Q&A] [answered]
Using Flyway to do migrations that produce a `FlywaySchemaHistory` table, this particular table would not be used in the application and would like to avoid generating the types file.

Having read through the docs it seems there is no current support for something of this nature. Have I missed some cli argument?

**Comment by kristiandupont** [comment, marked as answer]
You can use [typeFilter](https://kristiandupont.github.io/kanel/configuring.html#typefilter), which is perhaps poorly named since you are not the first to miss it. It takes a predicate, so you should be able to filter out anything you don't want.

**Reply by karloluis** [reply]
If anything it would help to have examples. It's still unclear how to make use of it. I'll give `console.log` my way into it I suppose.

**Reply by kristiandupont** [reply]
That's a good point. The only thing I can think of at the moment is the type filter included in the Knex extension:
https://github.com/kristiandupont/kanel/blob/main/packages/kanel-knex/src/knexTypeFilter.ts

Let me know if you need help
